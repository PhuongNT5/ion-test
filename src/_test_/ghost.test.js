import React from "react";
import { shallow, configure} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { create} from 'react-test-renderer'
configure({adapter: new Adapter()});
// describle('test button', function() {
  test('it shows the expected text when clicked', function() {
    function Button(props) {
      return <button>Nothing to do for now</button>;
    }
    
    const button = create(<Button/>);
    expect(button.toJSON()).toMatchSnapshot();
    // const clickFn = jest.fn();
    // const btn = shallow(
    //   <button handleClick={clickFn}>Start</button>
    // )
    // btn.simulate('click');
    // expect(clickFn).toHaveBeenCalled();

  })
// })
