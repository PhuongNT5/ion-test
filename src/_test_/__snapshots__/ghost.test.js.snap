// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it shows the expected text when clicked 1`] = `
<button>
  Nothing to do for now
</button>
`;
